catalogue_pipeline/id_minter/src/test/scala/uk/ac/wellcome/platform/idminter/steps/IdEmbedderTests.scala

```diff
@@ -23,7 +23,7 @@ class IdEmbedderTests
     with BeforeAndAfterEach
     with MockitoSugar
     with JsonTestUtil
-    with PatienceConfiguration {
+    with ExtendedPatience {
```

Comment:
If you're replacing PatienceConfiguration with ExtendedPatience, are you going to remove the patienceConfig below?

Reply:
oh thanks